command.go

```diff
@@ -22,10 +23,11 @@ type ResponseHandler func(args []interface{}) (interface{}, error)
 // Cmd stores the registered information about a command to return it later
 // when request by a command execution
 type Cmd struct {
-	Name      string        // Name of the command
-	Args      []interface{} // Arguments of the command
-	Responses []Response    // Slice of returned responses
-	Called    bool          // State for this command called or not
+	Name       string        // Name of the command
+	Args       []interface{} // Arguments of the command
+	Responses  []Response    // Slice of returned responses
+	Called     bool          // State for this command called or not
+	calledOnce sync.Once     // used to initialize Called
```

Comment:
Following a similar comment from the [other PR](https://github.com/rafaeljusto/redigomock/pull/59#discussion_r521967070), if we decide to break backward compatibility, we could change `Called` to a private attribute and add a `Called() bool` method just to make sure this is read-only.